src/main/resources/rs117/hd/lighting/lights.json

```diff
@@ -20947,15 +20947,18 @@
     "duration": 0.0,
     "range": 20.0,
     "objectIds": [
-      11474,
       196,
       197,
       "OIL_LAMP",
-      4501,
       7432,
       "CANDLES",
       "CANDLES_203",
+      "HARP",
```

Comment:
a harp casts light? :joy:

Reply:
not sure if that was intended or not

Reply:
Interesting the Json must of generated that